velox/functions/prestosql/benchmarks/JsonFileReader.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
```

Comment:
- Use make header-fix to add license headers to all new files.
- Use make format-fix to fix formatting issues.